src/components/CitySubTitle/CitySubTitle.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import Row from "react-bootstrap/Row";
+import "./CitySubTitle.css";
+
+function CitySubTitle({ citysubtitle }) {
+  return (
+    <Row className="city-sub-title">
```

Comment:
This should be changed to `CitySubTitle` to be more consistent with other components